sql/driver/wire.go

```diff
@@ -80,14 +100,17 @@ func (d Datum) Value() (driver.Value, error) {
 		val = t.BytesVal
 	case *Datum_StringVal:
 		val = t.StringVal
+	case *Datum_DateVal:
+		val = Date{t.DateVal.GoTime()}
```

Comment:
AFAIK, we never print timestamps with the time zone stripped. Any user that `Scan`s a timestamp into a `time.Time` and prints it with the time zone stripped is asking for trouble.